ui/src/tempVars/components/MetaQueryTemplateBuilder.tsx

```diff
@@ -115,7 +111,7 @@ class CustomMetaQueryTemplateBuilder extends PureComponent<
   }
 
   private executeQuery = async (): Promise<void> => {
-    const {template, source, onUpdateTemplate} = this.props
+    const {template, templates, source, onUpdateTemplate} = this.props
```

Comment:
Could be helpful to check in the future if the meta queries template values are in a list of nonNestedTemplates. One might write `SHOW TAG KEYS ON :currentdb: from :measurement:` where `:measurement:` is something like `SHOW MEASUREMENTS ON :currentdb:` and `:currentdb:` is just `SHOW DATABASES`.

 Multiple nesting will work fine at first, but will break when the app reloads because the template query will include an unsubstituted template value like `SHOW MEASUREMENT ON telegraf FROM :measurement:`